homeassistant/components/launch_library/sensor.py

```diff
@@ -120,6 +121,19 @@ class LaunchLibrarySensorEntityDescription(
             "description": sl.mission.description,
         },
     ),
+    LaunchLibrarySensorEntityDescription(
+        key="starship_event",
+        icon="mdi:calendar",
+        name="Next Starship event",
+        device_class=SensorDeviceClass.TIMESTAMP,
+        value_fn=lambda e: parse_datetime(e.date),
+        attributes_fn=lambda e: {
+            "title": e.name,
+            "location": e.location,
+            "stream": e.video_url,
+            "description": e.description,
```

Comment:
```suggestion
        value_fn=lambda se: parse_datetime(se.date),
        attributes_fn=lambda se: {
            "title": se.name,
            "location": se.location,
            "stream": se.video_url,
            "description": se.description,
```

Since we use 2 letters for the rest let's keep that up here :+1: